.lintr

```diff
@@ -1,7 +1,6 @@
-linters: with_defaults(
+linters: linters_with_defaults(
```

Comment:
IDE was complaining about this so had to change it, this is the updated version or something like that apparently